diem-move/diem-framework/experimental/sources/NFTGallery.move

```diff
@@ -0,0 +1,122 @@
+module 0x1::NFTGallery {
+    use Std::Errors;
+    use Std::GUID;
+    use 0x1::NFT::{Self, Token};
+    use Std::Option::{Self, Option};
+    use Std::Signer;
+    use Std::Vector;
+
+    /// Gallery holding tokens of `TokenType` as well as information of approved operators.
+    struct NFTGallery<phantom TokenType: store> has key {
+        gallery: vector<Token<TokenType>>
+    }
+
+    const MAX_U64: u128 = 18446744073709551615u128;
+    // Error codes
+    const EID_NOT_FOUND: u64 = 0;
+    const EBALANCE_NOT_PUBLISHED: u64 = 1;
+    const EBALANCE_ALREADY_PUBLISHED: u64 = 2;
+    const EINVALID_AMOUNT_OF_TRANSFER: u64 = 3;
+    const EALREADY_IS_OPERATOR: u64 = 4;
+    const ENOT_OPERATOR: u64 = 5;
+    const EINVALID_APPROVAL_TARGET: u64 = 6;
+
+    /// Add a token to the owner's gallery. If there is already a token of the same id in the
+    /// gallery, we combine it with the new one and make a token of greater value.
+    public fun add_to_gallery<TokenType: store>(owner: address, token: Token<TokenType>)
+    acquires NFTGallery {
```

Comment:
This seems weird, one line? (or at like the last function in this file).  Not sure if we have a formatter for move.